checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -198,6 +248,14 @@ private boolean isArrayLenZeroFieldAccess(ExpressionTree argument) {
         return false;
     }
 
+    private static boolean isLiteralZero(ExpressionTree tree) {
```

Comment:
Note the hacky implementation [here](https://github.com/eisop/checker-framework/blob/f1f8697b4c1e88f5edddb3bee9dea0b0b7f8c262/checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java#L167).
It would be good to use the same logic in both places.

More generally, maybe more logic can be reused?